tests/LoadCache.txt

```diff
@@ -1,5 +1,5 @@
 loadCache :: GenHaxl u ()
 loadCache = do
-  cacheRequest (CountAardvarks "yyy") (except (LogicError (NotFound "yyy")))
   cacheRequest (CountAardvarks "xxx") (Right (3))
```

Comment:
The order in which the cacheRequests are listed is changed by 3f64d4800ddad48aa08aa63c74396c5ce498da4a,
so I changed this file to match with what I get. As far as I can tell, the order is not relevant, is it?